components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -561,11 +561,10 @@ void BraveProfileSyncServiceImpl::OnResolvedSyncRecords(
     }
     // Send records to syncer
     if (get_record_cb_) {
-      sync_thread_->task_runner()->PostTask(
-          FROM_HERE,
-          base::BindOnce(&DoDispatchGetRecordsCallback,
-                         std::move(get_record_cb_),
-                         std::move(pending_received_records_)));
+      backend_task_runner_->PostTask(
```

Comment:
cc @darkdh

Reply:
checked upstream changes. There is no unsafe access issue